README.md

```diff
@@ -1,9 +1,27 @@
 # Northcoders News API
 
-## Background
+## About
 
-This repository mimics a real world backend service (such as reddit) which should provide this information to the front end architecture. It uses PSQL and interacts with it using [node-postgres](https://node-postgres.com/).
+Welcome to ncnews. This repository mimics a real world backend service (such as reddit) which should provide this information to the front end architecture. It uses PSQL and interacts with it using [node-postgres](https://node-postgres.com/).
```

Comment:
-don't sell yourself short by saying 'mimics a real world...' - just make the comparison (something like 'provides similar features to reddit')